aten/src/ATen/native/native_functions.yaml

```diff
@@ -8457,29 +8457,29 @@
   device_check: NoCheck   # TensorIterator
   variants: method, function
   dispatch:
-    CPU, CUDA: __lshift__
+    CPU, CUDA, MPS: __lshift__
```

Comment:
Just curious, where is the magic that ties `__lshift__` to `bitwise_left_shift_out`

Reply:
Here:
https://github.com/pytorch/pytorch/blob/3d7c424a750a1115bfbe56baf0f03b8a05e6ffe9/aten/src/ATen/native/BinaryOps.cpp#L1292